content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -4,17 +4,18 @@ reviewers:
 title: Upgrading kubeadm clusters
 content_type: task
 weight: 20
-min-kubernetes-server-version: 1.18
+min-kubernetes-server-version: 1.19
 ---
 
 <!-- overview -->
 
 This page explains how to upgrade a Kubernetes cluster created with kubeadm from version
-1.17.x to version 1.18.x, and from version 1.18.x to 1.18.y (where `y > x`).
+1.18.x to version 1.19.x, and from version 1.19.x to 1.19.y (where `y > x`).
 
 To see information about upgrading clusters created using older versions of kubeadm,
 please refer to following pages instead:
 
+- [Upgrading kubeadm cluster from 1.17 to 1.18](https://v1-18.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
```

Comment:
@rosti , Would you check the last two links. Are both links required? The links point to the 1.15 snapshot version of the docs.
1.18-1.19
1.17-1.18
1.16-1.17
1.15-1.1.6
1.14-1.15 ?

Reply:
The idea there is to keep as much as possible from the older upgrade docs to allow folks with older clusters to incrementally upgrade to modern Kubernetes versions.
These are good to be kept as long as the 1.15 snapshot is still present. If not, I can send a PR to remove the link.

Reply:
OK.